File: Java/백준/Bronze/1934. 최소공배수/최소공배수.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int n = sc.nextInt();
        
        for (int i = 0; i < n; i++) {
            int x = sc.nextInt();
            int y = sc.nextInt();
            
            int temp_x = x, temp_y = y;
            
            while (temp_y != 0) {
                int temp = temp_y;
                temp_y = temp_x % temp_y;
                temp_x = temp;
            }
            
            int lcm = x / temp_x * y;
            
            System.out.println(lcm);
        }
        
        sc.close();
    }
}

File: C++17/백준/Bronze/1934. 최소공배수/최소공배수.cc
#include <iostream>
using namespace std;

int gcd(int a, int b) {
    while (b != 0) {
        int temp = b;
        b = a % b;
        a = temp;
    }
    return a;
}

int main() {
    int n;
    cin >> n;
    
    for (int i = 0; i < n; i++) {
        int a, b;
        cin >> a >> b;

        int lcm = a / gcd(a, b) * b;
        cout << lcm << endl;
    }

    return 0;
}